v3/add_new_column_to_a_materialized_view_v3/README.md

```diff
@@ -1,3 +1,76 @@
-# Tinybird Versions - {{ YOUR USE CASE NAME HERE }}
+# Tinybird Versions - Add Column to a Materialized View using Releases
 
-Work in progress ...
+Introducing a new column to a Materialized View Data Source is a delicate process that needs to be handled with care to ensure data integrity and continuity. This guide will take you through the steps to achieve this without disrupting your data flow.
+
+> Remember to follow the [instructions](../README.md) to setup your Tinybird Data Project before jumping into the use-case steps
+
+This change needs to re-create the Materialized View and populate it again with all the data without stoping our ingestion.
+
+For that the steps will be:
+
+(In this Pull Request)
+1. Change the Materialized View (Pipe and Data Source) to add the new column.
+2. Bump version from 0.0.0 -> 0.0.1. It will create a new Preview Release internally forking the Materialized View and its dependencies.
+
+(Out of this Pull Request)
```

Comment:
Let's include it in the README.md so it's reproducible

Reply:
Changed, you can review again ;)